stories/storyshots.test.js

```diff
@@ -0,0 +1,4 @@
+import initStoryshots from 'storyshots';
+
+// Turn the Storybook stories into Jest snapshots
+initStoryshots();
```

Comment:
I'm not sure I understand the question? 

[Storyshots](https://github.com/storybooks/storyshots) generates the Jest snapshots automatically for us. So we don't have to create snapshots tests for each and every component ourselves.